sdk/tests/conformance2/rendering/framebuffer-texture-level1.html

```diff
@@ -43,13 +43,11 @@
 var wtu = WebGLTestUtils;
 var gl;
 
-function testFramebufferTextureWithNonZeroBaseLevel() {
+function testFramebufferTextureWithNonZeroBaseLevel(level) {
```

Comment:
Do you want to add an assertion that level > 1? Because otherwise someone else may come in here and find that the test fails because the texture is actually complete if called with level=0.